test/cpp/aot_inductor/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup
+from torch.utils.cpp_extension import BuildExtension, CppExtension
+import os 
+
+inductor_module = CppExtension(
```

Comment:
feedback from @albanD : just use load_inline() function instead of using a full module